test/test_ftp_object.jl

```diff
@@ -1,171 +1,325 @@
-using FTPClient
+@testset "FTPObject" begin
+
+    expected_header_port = r"229 Entering Extended Passive Mode \(\|\|\|\d*\|\)"
```

Comment:
This regular expression is used all over the place. Consider making it a global constant.